### ISSUE TITLE
March 01 update broke blueprint mode

### ISSUE THREAD

**Issue by dejayc** [sirskunkalot/PlanBuild#41]
The frost-cave updates (made public on March 01, 2022) seem to have introduced a change that breaks blueprint mode, with the following error message reported in the console:

```
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type Console and name InputText and parameters
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static void TerrainTools.Patches.Commands::Postfix(Console& __instance)
Stack trace: HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)                                       
Rethrow as HarmonyException: Patching exception in method null                                                          
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)                                                                                    
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)                                       
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)                       
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)                                                                               
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)             
TerrainTools.TerrainTools.Awake () (at <a0e26b0a4c024ac38bb1d83b45b5e607>:0)                                            
UnityEngine.GameObject:AddComponent(Type)                                                                               
BepInEx.Bootstrap.Chainloader:Start()                                                                                   
UnityEngine.Application:.cctor()
```

Practically speaking, equipping the blueprint rune shows the following text string in the bottom-right of the screen:

```
RuneModeToggle!marcopogo.PlanBuild Switch to Blueprint mode
```

Jotunn was also updated on this day.

**Comment by sirskunkalot** [comment]
TerrainTools from the error stack is not part of PlanBuild. Do you have other mods installed as well? There are some popular mods that broke with the update.

**Comment by Xeno1177** [comment]
Same issue.  This shows in the bottom right and planbuild is the only thing installed.

RuneModeToggle!marcopogo.PlanBuild Switch to Blueprint mode

Pressing "P" does nothing.  Changing the hotkey has no effect.  If I wipe the install and boot straight into the map, it will work sometimes.  Upon next boot, it shows the above message and refuses to switch into blurprint mode.

**Comment by sirskunkalot** [comment]
Should already be fixed with 0.9.4 - I forgot to release a fixed version when the cave update dropped.
